### PR TITLE
SE-0193: @inlinable implies @usableFromInline

### PR DESCRIPTION
**Pull request by slavapestov**
This was a minor change made to the proposal by the core team after it was accepted.

### CODE REVIEW

**Comment by slavapestov** [comment]
@swift-ci Please test

**Comment by slavapestov** [comment]
@swift-ci Please test source compatibility

**Comment by slavapestov** [comment]
@swift-ci Please benchmark

File: CHANGELOG.md [hunk 1]
```diff
@@ -61,6 +61,18 @@ Swift 5.0
 Swift 4.2
 ---------
 
+* [SE-0193][]
```
Comment (DougGregor) [inline CHANGELOG.md]: Thank you for remembering the change log!

**Review by DougGregor — APPROVED** [review]

**Pull request merged**